### ISSUE TITLE
No edges when outline effect is combined with Fog

### ISSUE THREAD

**Issue by fedorvaneldijk** [mrdoob/three.js#16463]
When using Fog in combination with the OutlineEffect, the outline effect does not render any edges. @Mugen87 looked into this and came with this response:

```
This looks like a bug. I’ve tested with the webgl_loader_mmd example and get the following runtime error when using fog.

THREE.WebGLShader: Shader couldn’t compile.
three.js:16878 THREE.WebGLShader: gl.getShaderInfoLog() vertex ERROR: 0:202: ‘fogDepth’ : redefinition

Please file an issue at github. This needs to be fixed in OutlineEffect.
```

##### Three.js version

- [ ] Dev
- [X ] r104
- [ X] r103

##### Browser

- [x] All of them
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [x] All of them
- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

##### Hardware Requirements (graphics card, VR Device, ...)

nothing special, problem on all devices...

**Comment by Mugen87** [comment]
see https://discourse.threejs.org/t/how-to-combine-fog-with-outline-effect/7577

**Comment by Mugen87** [comment]
Solved via #16464
